tutorials/adapt-vqe/adapt-vqe.ipynb

```diff
@@ -0,0 +1,1640 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    def cholesky(V, eps):
</pre><p>Is this significantly different than the version in Numpy (np.linalg.cholesky)? If the answer is no, then I would just go with Numpy's implementation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yeah, it's based on code that Mario wrote for the chemistry VQE tutorials which I think he used in [this arXiv paper](https://arxiv.org/abs/1808.02625). The numpy version throws an error if I try to use it.

Reply:
This is an implementation of the "modified Cholesky" decomposition which differs from the standard Cholesky decomposition in that it only returns a small list of vectors needed to reconstruct the matrix, rather than an entire matrix. ffsim has an implementation documented here: https://qiskit-community.github.io/ffsim/api/ffsim.linalg.html#ffsim.linalg.modified_cholesky.